docs/static/js/12.0f058a3f.js

```diff
@@ -1,2 +0,0 @@
-(window.webpackJsonp=window.webpackJsonp||[]).push([[12],{"./.docz/app/imports.js":function(n,e,i){"use strict";i.r(e),i.d(e,"imports",function(){return r});var r={"index.mdx":function(){return i.e(2).then(i.bind(null,"./index.mdx"))},"packages/accordion/src/index.mdx":function(){return Promise.all([i.e(0),i.e(4)]).then(i.bind(null,"./packages/accordion/src/index.mdx"))},"packages/accordion-multi/src/index.mdx":function(){return Promise.all([i.e(0),i.e(3)]).then(i.bind(null,"./packages/accordion-multi/src/index.mdx"))},"packages/forms/src/index.mdx":function(){return Promise.all([i.e(0),i.e(5)]).then(i.bind(null,"./packages/forms/src/index.mdx"))},"packages/image/src/index.mdx":function(){return Promise.all([i.e(0),i.e(6)]).then(i.bind(null,"./packages/image/src/index.mdx"))},"packages/list-inline/src/index.mdx":function(){return Promise.all([i.e(0),i.e(7)]).then(i.bind(null,"./packages/list-inline/src/index.mdx"))},"packages/paging/src/index.mdx":function(){return Promise.all([i.e(0),i.e(8)]).then(i.bind(null,"./packages/paging/src/index.mdx"))},"packages/render/src/index.mdx":function(){return i.e(9).then(i.bind(null,"./packages/render/src/index.mdx"))},"packages/tabs/src/index.mdx":function(){return Promise.all([i.e(0),i.e(10)]).then(i.bind(null,"./packages/tabs/src/index.mdx"))}}}},0,[0,3,4,5,6,7,8,10,2,9]]);
```

Comment:
Why are you committing static files in the 'docs' folder? I would update the `.gitignore` file to exclude these.

Reply:
The git page is set up to read from the docs folder to generate the webpage. I believe the static files are necessary for this.